src/nextjournal/clerk/config.clj

```diff
@@ -19,7 +22,9 @@
 
 (defonce !resource->url
   (atom (or resource-manifest-from-props
-            default-resource-manifest)))
+            ;; assume that CI will have published a CAS-link under this lookup,
+            ;; prior to hitting this code-path
+            (edn/read-string (slurp lookup-url)))))
```

Comment:
The one thing I'm struggling with a bit here, is that it's pretty opaque what's inside the lookup file now (without running curl against the lookup-url, that is). Wondering if there's a way for us to get some of the clarify of `default-resource-manifest` back.

Reply:
I'll add a comment.